code/modules/food_and_drinks/food/foods/pizza.dm

```diff
@@ -252,8 +374,9 @@
 		return
 	..()
 
+
 /obj/item/pizzabox/margherita/Initialize(mapload)
-	pizza = new /obj/item/reagent_containers/food/snacks/sliceable/pizza/margherita(src)
+	pizza = new /obj/item/reagent_containers/food/snacks/sliceable/pizza/margheritapizza(src)
 	box_tag = "margherita deluxe"
 	return ..()
```

Comment:
Initialize has the parent call first, not last.

Please convert
```dm
/obj/item/pizzabox/margherita/Initialize(mapload)
	pizza = new /obj/item/reagent_containers/food/snacks/sliceable/pizza/margheritapizza(src)
	box_tag = "margherita deluxe"
	return ..()
```

to
```dm
/obj/item/pizzabox/margherita/Initialize(mapload)
	. = ..()
	pizza = new /obj/item/reagent_containers/food/snacks/sliceable/pizza/margheritapizza(src)
	box_tag = "margherita deluxe"
```

And do the same for the others.